projects/components/facet/bootstrap/facet-tree/facet-tree.ts

```diff
@@ -21,10 +21,12 @@ export class BsFacetTree extends AbstractFacet implements OnChanges, OnDestroy {
     @Input() showCount: boolean = true; // Show the number of occurrences
     @Input() allowExclude: boolean = true; // Allow to exclude selected items
     @Input() allowOr: boolean = true; // Allow to search various items in OR mode
+    @Input() allowAnd: boolean = true; // Allow to search various items in OR mode
```

Comment:
can this be `false` by default? (consistent w/ line 164)

Reply:
you're right. Should be false by default (treepath facet for example should not "allow And")